modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/components/__tests__/__snapshots__/LineChart.jsx.snap

```diff
@@ -1,6 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LineChart should get short intervals 1`] = `<div />`;
+exports[`LineChart should get short intervals 1`] = `
```

Comment:
@interaminense I believe these snaps update are related to that Recharts update we had to do